massgov/mayflower
# Patternlab/DP-13939  front end for contextual login

## Description
Updates utility nav to include contextual log in links and move markup into precontent for page editors

## Related Issue / Ticket

- [https://jira.mass.gov/browse/DP-13939](DP-13939)

## Steps to Test


1. From any page, click the utility nav login link to see the extra links in the dropdown menu

## Screenshots
<img width="1407" alt="Screen Shot 2019-06-16 at 4 47 55 PM" src="https://user-images.githubusercontent.com/18662734/59569974-85ed0100-9056-11e9-8d17-c9b4b1b14c54.png">
